src/applications/vaos/tests/appointment-list/components/RequestedAppointmentsList.unit.spec.jsx

```diff
@@ -311,101 +320,63 @@ describe('VAOS <RequestedAppointmentsList> with the VAOS service', () => {
 
   it('should show cc request', async () => {
     const startDate = moment.utc();
-    const appointment = getVAOSRequestMock();
```

Comment:
I don't think we want to change the existing tests. That code is still going to be the production code for quite a while, I don't think we want to have it go untested. We should be adding new tests (which don't need to necessarily duplicate all the current tests, but do at least test the new transformer and api call are integrated with the page components).

I think you can copy the tests you updated to be new tests and restore the old ones from master.

Reply:
I think this might be due to a confusing diff from an outdated commit. The new tests are under the `'VAOS <RequestedAppointmentsList> with the VAOS service'` describe block and the existing tests under the `'VAOS <RequestedAppointmentsList>'` were left unchanged.

Reply:
Ah, gotcha, yeah, not sure how I got to that diff instead of the regular one.